observability/interceptor.go

```diff
@@ -43,17 +45,17 @@ func (i *Interceptor) WrapUnary(next connect.UnaryFunc) connect.UnaryFunc {
 
 		resp, err := next(ctx, req)
 		if err != nil {
+			// Record FTL errors for non-wire errors.
+			if !connect.IsWireError(err) {
+				recordVerbCallError(ctx, req.Header())
+			}
```

Comment:
@alecthomas 3 things here:
* Hopefully, this is what you were thinking for the "non-transport" error check. Happy to update if you have something else in mind
* At this point, I don't think we have the headers to determine which `verb` had an error, so we might need to figure out a different way/place to record these, or perhaps we can resolve the header issue
* Technically, `recordVerbCallError` can fail with errors, but since I'm already in an error block I'm ignoring the new error if it occurs. Is there a better way in `go` to handle that?

Reply:
We might need an interceptor to add response headers when a call fails.